jkube-kit/doc/src/main/asciidoc/inc/remote-dev/_index.adoc

```diff
@@ -0,0 +1,178 @@
+:remote-dev-goal: {goal-prefix}:remote-dev
+ifeval::["{plugin-type}" == "gradle"]
+:remote-dev-goal: {task-prefix}RemoteDev
+endif::[]
+
+[[jkube:remote-dev]]
+// TODO: Fix indentation problems due to misalignment with TOC headings
+ifeval::["{plugin-type}" == "maven"]
+== *{remote-dev-goal}* (preview)
+endif::[]
+ifeval::["{plugin-type}" == "gradle"]
+=== *{remote-dev-goal}* (preview)
+endif::[]
+
+The Eclipse JKube Remote Development goal allows you to run and debug code in your *local* machine:
```

Comment:
Maybe this line should be moved in if else block
```suggestion
The Eclipse JKube Remote Development (goal|task) allows you to run and debug code in your *local* machine:
```

Reply:
I'll rephrase